Mathlib/Data/Sym/Sym2.lean

```diff
@@ -416,6 +416,7 @@ theorem map_id' : (map fun x : α => x) = id :=
 
 /-! ### Diagonal -/
 
+variable {e : Sym2 α} {f : α → β}
```

Comment:
The rest of this file seems to use z not e

Reply:
Both are used. But `z` is also used as an element of `α` so I think `e` is a bit better.